skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -634,10 +638,10 @@ def _fast_nl_means_denoising_2d(image, int s=7, int d=13, double h=0.1,
     # Normalize pixel values using sum of weights of contributing patches
     for row in range(offset, n_row - offset):
         for col in range(offset, n_col - offset):
-            for channel in range(n_ch):
+            for ch in range(n_ch):
```

Comment:
Why this change? :)

Reply:
I should have probably left it as channels as that is more explicit.  I have now updated to use `channels` and `n_channels` rather than `ch`, `n_ch` everywhere.